src/pyodide/internal/metadata.ts

```diff
@@ -39,5 +39,6 @@ export const TRANSITIVE_REQUIREMENTS =
 
 // Entrypoints
 export const MAIN_MODULE_NAME = MetadataReader.getMainModule();
+// TODO: Remove the below.
```

Comment:
Can you explain the reason they should be removed here? Not obvious to me.

Reply:
Why not delete them instead of adding a TODO?

Reply:
Now done